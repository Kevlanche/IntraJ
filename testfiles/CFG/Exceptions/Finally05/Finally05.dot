digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[6]"; "Opt[9]"; "List[10]"; "List[11]"}
{ rank = same; "List[7]"; "MethodDecl[12]"}
{ rank = same; "Modifier[8]"; "Entry[13]"; "Modifiers[14]"; "PrimitiveTypeAccess[17]"; "List[18]"; "List[19]"; "Opt[20]"; "Exit[21]"}
{ rank = same; "List[15]"; "Block[21]"}
{ rank = same; "Modifier[16]"; "List[22]"}
{ rank = same; "WhileStmt[23]"; "ExprStmt[84]"}
{ rank = same; "BooleanLiteral[24]"; "Block[25]"; "Dot[85]"}
{ rank = same; "List[26]"; "Dot[86]"; "MethodAccess[89]"}
{ rank = same; "ExprStmt[27]"; "TryStmt[38]"; "TypeAccess[87]"; "VarAccess[88]"; "List[91]"; "List[93]"; "List[94]"; "UncheckedExceptions[90]"}
{ rank = same; "Dot[28]"; "Block[39]"; "List[68]"; "Opt[69]"; "UncheckedExceptions[83]"; "StringLiteral[92]"}
{ rank = same; "Dot[29]"; "MethodAccess[32]"; "List[40]"; "Block[70]"}
{ rank = same; "TypeAccess[30]"; "VarAccess[31]"; "List[34]"; "List[36]"; "List[37]"; "UncheckedExceptions[33]"; "VarDeclStmt[41]"; "BreakStmt[49]"; "List[71]"}
{ rank = same; "StringLiteral[35]"; "Modifiers[42]"; "PrimitiveTypeAccess[44]"; "List[45]"; "List[67]"; "UncheckedExceptions[50]"; "ExprStmt[72]"}
{ rank = same; "List[43]"; "VariableDeclarator[46]"; "NTAFinallyBlock[52]"; "Dot[73]"}
{ rank = same; "List[47]"; "Opt[48]"; "List[53]"; "Dot[74]"; "MethodAccess[77]"}
{ rank = same; "Block[54]"; "TypeAccess[75]"; "VarAccess[76]"; "List[79]"; "List[81]"; "List[82]"; "UncheckedExceptions[78]"}
{ rank = same; "List[55]"; "StringLiteral[80]"}
{ rank = same; "ExprStmt[56]"}
{ rank = same; "Dot[57]"}
{ rank = same; "Dot[58]"; "MethodAccess[61]"}
{ rank = same; "TypeAccess[59]"; "VarAccess[60]"; "List[63]"; "List[65]"; "List[66]"; "UncheckedExceptions[62]"}
{ rank = same; "StringLiteral[64]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Finally05 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[6]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[7]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[8]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[9]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[10]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[11]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[12]" [label="MethodDecl\n < fun >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[13]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[14]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[15]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[16]" [label="Modifier\n < static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[17]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[18]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[19]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[20]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[21]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[22]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"WhileStmt[23]" [label="WhileStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BooleanLiteral[24]" [label="BooleanLiteral\n < true >", fillcolor=white   style=filled  ]
"Block[25]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[27]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[28]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[29]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[30]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[31]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[32]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[35]" [label="StringLiteral\n < InWhile >", fillcolor=white   style=filled  ]
"List[36]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[33]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TryStmt[38]" [label="TryStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[39]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[41]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[42]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[43]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[44]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[45]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[46]" [label="VariableDeclarator\n < int f >", fillcolor=white   style=filled  ]
"List[47]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[48]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"BreakStmt[49]" [label="BreakStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"NTAFinallyBlock[52]" [label="NTAFinallyBlock\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[53]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[54]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[55]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[56]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[57]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[58]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[59]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[60]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[61]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[63]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[64]" [label="StringLiteral\n < InFinally >", fillcolor=white   style=filled  ]
"List[65]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[66]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[62]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[67]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[50]" [label="UncheckedExceptions\n ", fillcolor=white   style=filled  ]
"List[68]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[69]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[70]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[71]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[72]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[73]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[74]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[75]" [label="TypeAccess\n < System >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarAccess[76]" [label="VarAccess\n < out >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodAccess[77]" [label="MethodAccess\n < println >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[79]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[80]" [label="StringLiteral\n < InFinally >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[81]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[82]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[78]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[83]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[84]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[85]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Dot[86]" [label="Dot\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"TypeAccess[87]" [label="TypeAccess\n < System >", fillcolor=white   style=filled  ]
"VarAccess[88]" [label="VarAccess\n < out >", fillcolor=white   style=filled  ]
"MethodAccess[89]" [label="MethodAccess\n < println >", fillcolor=white   style=filled  ]
"List[91]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"StringLiteral[92]" [label="StringLiteral\n < AfterWhile >", fillcolor=white   style=filled  ]
"List[93]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[94]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"UncheckedExceptions[90]" [label="UncheckedExceptions\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Exit[21]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[7]" -> "Modifier[8]" [style=dashed, color=gray]
"Modifiers[6]" -> "List[7]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[9]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[10]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Entry[13]" [style=dashed, color=gray]
"List[15]" -> "Modifier[16]" [style=dashed, color=gray]
"Modifiers[14]" -> "List[15]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Modifiers[14]" [style=dashed, color=gray]
"MethodDecl[12]" -> "PrimitiveTypeAccess[17]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[18]" [style=dashed, color=gray]
"MethodDecl[12]" -> "List[19]" [style=dashed, color=gray]
"WhileStmt[23]" -> "BooleanLiteral[24]" [style=dashed, color=gray]
"Dot[29]" -> "TypeAccess[30]" [style=dashed, color=gray]
"Dot[29]" -> "VarAccess[31]" [style=dashed, color=gray]
"Dot[28]" -> "Dot[29]" [style=dashed, color=gray]
"List[34]" -> "StringLiteral[35]" [style=dashed, color=gray]
"MethodAccess[32]" -> "List[34]" [style=dashed, color=gray]
"MethodAccess[32]" -> "List[36]" [style=dashed, color=gray]
"MethodAccess[32]" -> "List[37]" [style=dashed, color=gray]
"MethodAccess[32]" -> "UncheckedExceptions[33]" [style=dashed, color=gray]
"Dot[28]" -> "MethodAccess[32]" [style=dashed, color=gray]
"ExprStmt[27]" -> "Dot[28]" [style=dashed, color=gray]
"List[26]" -> "ExprStmt[27]" [style=dashed, color=gray]
"Modifiers[42]" -> "List[43]" [style=dashed, color=gray]
"VarDeclStmt[41]" -> "Modifiers[42]" [style=dashed, color=gray]
"VarDeclStmt[41]" -> "PrimitiveTypeAccess[44]" [style=dashed, color=gray]
"VariableDeclarator[46]" -> "List[47]" [style=dashed, color=gray]
"VariableDeclarator[46]" -> "Opt[48]" [style=dashed, color=gray]
"List[45]" -> "VariableDeclarator[46]" [style=dashed, color=gray]
"VarDeclStmt[41]" -> "List[45]" [style=dashed, color=gray]
"List[40]" -> "VarDeclStmt[41]" [style=dashed, color=gray]
"Dot[58]" -> "TypeAccess[59]" [style=dashed, color=gray]
"Dot[58]" -> "VarAccess[60]" [style=dashed, color=gray]
"Dot[57]" -> "Dot[58]" [style=dashed, color=gray]
"List[63]" -> "StringLiteral[64]" [style=dashed, color=gray]
"MethodAccess[61]" -> "List[63]" [style=dashed, color=gray]
"MethodAccess[61]" -> "List[65]" [style=dashed, color=gray]
"MethodAccess[61]" -> "List[66]" [style=dashed, color=gray]
"MethodAccess[61]" -> "UncheckedExceptions[62]" [style=dashed, color=gray]
"Dot[57]" -> "MethodAccess[61]" [style=dashed, color=gray]
"ExprStmt[56]" -> "Dot[57]" [style=dashed, color=gray]
"List[55]" -> "ExprStmt[56]" [style=dashed, color=gray]
"Block[54]" -> "List[55]" [style=dashed, color=gray]
"List[53]" -> "Block[54]" [style=dashed, color=gray]
"NTAFinallyBlock[52]" -> "List[53]" [style=dashed, color=gray]
"BreakStmt[49]" -> "NTAFinallyBlock[52]" [style=dashed, color=gray]
"BreakStmt[49]" -> "List[67]" [style=dashed, color=gray]
"BreakStmt[49]" -> "UncheckedExceptions[50]" [style=dashed, color=gray]
"List[40]" -> "BreakStmt[49]" [style=dashed, color=gray]
"Block[39]" -> "List[40]" [style=dashed, color=gray]
"TryStmt[38]" -> "Block[39]" [style=dashed, color=gray]
"TryStmt[38]" -> "List[68]" [style=dashed, color=gray]
"Dot[74]" -> "TypeAccess[75]" [style=dashed, color=gray]
"Dot[74]" -> "VarAccess[76]" [style=dashed, color=gray]
"Dot[73]" -> "Dot[74]" [style=dashed, color=gray]
"List[79]" -> "StringLiteral[80]" [style=dashed, color=gray]
"MethodAccess[77]" -> "List[79]" [style=dashed, color=gray]
"MethodAccess[77]" -> "List[81]" [style=dashed, color=gray]
"MethodAccess[77]" -> "List[82]" [style=dashed, color=gray]
"MethodAccess[77]" -> "UncheckedExceptions[78]" [style=dashed, color=gray]
"Dot[73]" -> "MethodAccess[77]" [style=dashed, color=gray]
"ExprStmt[72]" -> "Dot[73]" [style=dashed, color=gray]
"List[71]" -> "ExprStmt[72]" [style=dashed, color=gray]
"Block[70]" -> "List[71]" [style=dashed, color=gray]
"Opt[69]" -> "Block[70]" [style=dashed, color=gray]
"TryStmt[38]" -> "Opt[69]" [style=dashed, color=gray]
"TryStmt[38]" -> "UncheckedExceptions[83]" [style=dashed, color=gray]
"List[26]" -> "TryStmt[38]" [style=dashed, color=gray]
"Block[25]" -> "List[26]" [style=dashed, color=gray]
"WhileStmt[23]" -> "Block[25]" [style=dashed, color=gray]
"List[22]" -> "WhileStmt[23]" [style=dashed, color=gray]
"Dot[86]" -> "TypeAccess[87]" [style=dashed, color=gray]
"Dot[86]" -> "VarAccess[88]" [style=dashed, color=gray]
"Dot[85]" -> "Dot[86]" [style=dashed, color=gray]
"List[91]" -> "StringLiteral[92]" [style=dashed, color=gray]
"MethodAccess[89]" -> "List[91]" [style=dashed, color=gray]
"MethodAccess[89]" -> "List[93]" [style=dashed, color=gray]
"MethodAccess[89]" -> "List[94]" [style=dashed, color=gray]
"MethodAccess[89]" -> "UncheckedExceptions[90]" [style=dashed, color=gray]
"Dot[85]" -> "MethodAccess[89]" [style=dashed, color=gray]
"ExprStmt[84]" -> "Dot[85]" [style=dashed, color=gray]
"List[22]" -> "ExprStmt[84]" [style=dashed, color=gray]
"Block[21]" -> "List[22]" [style=dashed, color=gray]
"Opt[20]" -> "Block[21]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Opt[20]" [style=dashed, color=gray]
"MethodDecl[12]" -> "Exit[21]" [style=dashed, color=gray]
"List[11]" -> "MethodDecl[12]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[11]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[21]" -> "MethodAccess[61]" [color=red, constraint=false]
"Exit[21]" -> "MethodAccess[89]" [color=red, constraint=false]
"MethodAccess[89]" -> "Exit[21]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[89]" -> "StringLiteral[92]" [color=red, constraint=false]
"StringLiteral[92]" -> "MethodAccess[89]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[92]" -> "VarAccess[88]" [color=red, constraint=false]
"VarAccess[88]" -> "StringLiteral[92]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[88]" -> "TypeAccess[87]" [color=red, constraint=false]
"TypeAccess[87]" -> "VarAccess[88]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[87]" -> "BooleanLiteral[24]" [color=red, constraint=false]
"TypeAccess[87]" -> "MethodAccess[61]" [color=red, constraint=false]
"MethodAccess[61]" -> "TypeAccess[87]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[61]" -> "Exit[21]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[61]" -> "StringLiteral[64]" [color=red, constraint=false]
"StringLiteral[64]" -> "MethodAccess[61]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[64]" -> "VarAccess[60]" [color=red, constraint=false]
"VarAccess[60]" -> "StringLiteral[64]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[60]" -> "TypeAccess[59]" [color=red, constraint=false]
"TypeAccess[59]" -> "VarAccess[60]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[59]" -> "UncheckedExceptions[50]" [color=red, constraint=false]
"UncheckedExceptions[50]" -> "TypeAccess[59]" [color=blue, constraint=false, xlabel=" "] 
"UncheckedExceptions[50]" -> "VariableDeclarator[46]" [color=red, constraint=false]
"VariableDeclarator[46]" -> "UncheckedExceptions[50]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[46]" -> "MethodAccess[32]" [color=red, constraint=false]
"MethodAccess[32]" -> "VariableDeclarator[46]" [color=blue, constraint=false, xlabel=" "] 
"MethodAccess[32]" -> "StringLiteral[35]" [color=red, constraint=false]
"StringLiteral[35]" -> "MethodAccess[32]" [color=blue, constraint=false, xlabel=" "] 
"StringLiteral[35]" -> "VarAccess[31]" [color=red, constraint=false]
"VarAccess[31]" -> "StringLiteral[35]" [color=blue, constraint=false, xlabel=" "] 
"VarAccess[31]" -> "TypeAccess[30]" [color=red, constraint=false]
"TypeAccess[30]" -> "VarAccess[31]" [color=blue, constraint=false, xlabel=" "] 
"TypeAccess[30]" -> "BooleanLiteral[24]" [color=red, constraint=false]
"BooleanLiteral[24]" -> "TypeAccess[30]" [color=blue, constraint=false, xlabel="T "] 
"BooleanLiteral[24]" -> "TypeAccess[87]" [color=blue, constraint=false, xlabel="F "] 
"BooleanLiteral[24]" -> "Entry[13]" [color=red, constraint=false]
"Entry[13]" -> "BooleanLiteral[24]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"Modifiers[6]" -> "Opt[9]" -> "List[10]" -> "List[11]" [style=invis]
"Entry[13]" -> "Modifiers[14]" -> "PrimitiveTypeAccess[17]" -> "List[18]" -> "List[19]" -> "Opt[20]" -> "Exit[21]" [style=invis]
"WhileStmt[23]" -> "ExprStmt[84]" [style=invis]
"BooleanLiteral[24]" -> "Block[25]" [style=invis]
"ExprStmt[27]" -> "TryStmt[38]" [style=invis]
"Dot[29]" -> "MethodAccess[32]" [style=invis]
"TypeAccess[30]" -> "VarAccess[31]" [style=invis]
"Block[39]" -> "List[68]" -> "Opt[69]" [style=invis]
"VarDeclStmt[41]" -> "BreakStmt[49]" [style=invis]
"Modifiers[42]" -> "PrimitiveTypeAccess[44]" -> "List[45]" [style=invis]
"List[47]" -> "Opt[48]" [style=invis]
"Dot[58]" -> "MethodAccess[61]" [style=invis]
"TypeAccess[59]" -> "VarAccess[60]" [style=invis]
"Dot[74]" -> "MethodAccess[77]" [style=invis]
"TypeAccess[75]" -> "VarAccess[76]" [style=invis]
"Dot[86]" -> "MethodAccess[89]" [style=invis]
"TypeAccess[87]" -> "VarAccess[88]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
95", fillcolor=1]
Edges [label="Total Nr. Edges
130", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
17", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
18", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}