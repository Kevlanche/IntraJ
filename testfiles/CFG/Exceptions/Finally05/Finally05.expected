NodeName: Entry[13] Successors: [BooleanLiteral[24]]
NodeName: BooleanLiteral[24] Successors: [TypeAccess[30], TypeAccess[87]]
NodeName: TypeAccess[30] Successors: [VarAccess[31]]
NodeName: VarAccess[31] Successors: [StringLiteral[35]]
NodeName: StringLiteral[35] Successors: [MethodAccess[32]]
NodeName: MethodAccess[32] Successors: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Successors: [UncheckedExceptions[50]]
NodeName: UncheckedExceptions[50] Successors: [TypeAccess[59]]
NodeName: TypeAccess[59] Successors: [VarAccess[60]]
NodeName: VarAccess[60] Successors: [StringLiteral[64]]
NodeName: StringLiteral[64] Successors: [MethodAccess[61]]
NodeName: MethodAccess[61] Successors: [TypeAccess[87], Exit[21]]
NodeName: TypeAccess[87] Successors: [VarAccess[88]]
NodeName: VarAccess[88] Successors: [StringLiteral[92]]
NodeName: StringLiteral[92] Successors: [MethodAccess[89]]
NodeName: MethodAccess[89] Successors: [Exit[21]]
NodeName: Exit[21] Successors: []

NodeName: Exit[21] Predecessor: [MethodAccess[61], MethodAccess[89]]
NodeName: MethodAccess[61] Predecessor: [StringLiteral[64]]
NodeName: StringLiteral[64] Predecessor: [VarAccess[60]]
NodeName: VarAccess[60] Predecessor: [TypeAccess[59]]
NodeName: TypeAccess[59] Predecessor: [UncheckedExceptions[50]]
NodeName: UncheckedExceptions[50] Predecessor: [VariableDeclarator[46]]
NodeName: VariableDeclarator[46] Predecessor: [MethodAccess[32]]
NodeName: MethodAccess[32] Predecessor: [StringLiteral[35]]
NodeName: StringLiteral[35] Predecessor: [VarAccess[31]]
NodeName: VarAccess[31] Predecessor: [TypeAccess[30]]
NodeName: TypeAccess[30] Predecessor: [BooleanLiteral[24]]
NodeName: BooleanLiteral[24] Predecessor: [Entry[13]]
NodeName: Entry[13] Predecessor: []
NodeName: MethodAccess[89] Predecessor: [StringLiteral[92]]
NodeName: StringLiteral[92] Predecessor: [VarAccess[88]]
NodeName: VarAccess[88] Predecessor: [TypeAccess[87]]
NodeName: TypeAccess[87] Predecessor: [BooleanLiteral[24], MethodAccess[61]]
